src/metpy/calc/thermo.py

```diff
@@ -442,6 +442,108 @@ def _lcl_iter(p, p0, w, t):
     return lcl_p, globals()['dewpoint']._nounit(vapor_pressure._nounit(lcl_p, w))
 
 
+@exporter.export
+@preprocess_and_wrap()
+@check_units('[pressure]', '[temperature]', '[temperature]')
+def ccl(pressure, temperature, dewpoint, height=None, mixed_layer_depth=None, which='top'):
+    r"""Calculate the convective condensation level (CCL).
+
+    This function is implemented by simplifying the mixing ratio and
+    the saturation vapor pressure equation to
+
+    .. math:: A(P) = ln(\frac{rP}{6.112(r+\epsilon)}) = \frac{17.67T}{T+243.5}
```

Comment:
Do you have a citation we can use for this approach?

Reply:
I don't have a citation for it, as I can't seem to find an implementation online. Attached is a simple derivation. It would be nice if someone could double-check the math as well. 

![image](https://user-images.githubusercontent.com/38182337/190536449-e9ee1a05-91ea-4e08-a783-763f43497cef.png)

Reply:
After digging into the description in the USAF manual, I see this is straightforward.